pkg/token/execCredentialPlugin.go

```diff
@@ -28,6 +32,16 @@ type execCredentialPlugin struct {
 }
 
 func New(o *Options) (ExecCredentialPlugin, error) {
+	env := os.Getenv(execInfoEnv)
+	fmt.Fprintln(os.Stderr, os.Getenv(execInfoEnv))
```

Comment:
this line is for debugging, right? you can log it using klog.V(10).Info(), though

Reply:
besides, you already have `env`. don't need to `os.Getenv()` again?

Reply:
Addressed for next commit.